src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java

```diff
@@ -114,10 +114,6 @@ public InfernalExpansion() {
         // Registering Configs
         modLoadingContext.registerConfig(ModConfig.Type.CLIENT, ConfigHolder.CLIENT_SPEC);
         modLoadingContext.registerConfig(ModConfig.Type.COMMON, ConfigHolder.COMMON_SPEC);
-
-        // Baking Configs
-        ConfigHelper.bakeClient(null);
-        ConfigHelper.bakeCommon(null);
```

Comment:
What is this replaced by?

Reply:
No replacement should be necessary. ModConfigEvent fires on Config load and calls the bake methods, which is the earliest the config values can be retrieved. Nothing in our registry should rely on the config. I believe the earliest the values will be used is on datapack load, which happens after the config is loaded.

Reply:
I breakpointed all the config getters to double check this and, yes, it is safe to remove these as the ModConfigEvent will bake the configs at the time of load and we don't retrieve (and shouldn't anyway) any config values prior to that. So, when values are retrieved, the config will still always have been baked.